rules/internal/objc_provider_utils.bzl

```diff
@@ -55,9 +55,29 @@ def _merge_dynamic_framework_providers(ctx, dynamic_framework_providers):
 
     return apple_common.new_dynamic_framework_provider(**fields)
 
+def _merge_cc_info_providers(cc_info_providers):
+    fields = {}
+
+    # This is used in an isolated context - only the public ones
+    merge_keys = [
+        "headers",
+    ]
```

Comment:
do you envision this list of keys getting longer? if not, the code would be much more readable if you just access the attr directly.

Reply:
This function is designed to have a familiar API to rest of the "merge" functions in the module - which operate all on a list of keys - I the issue is that the key's aren't a parameter here but they are there.